splunk_add_on_ucc_framework/global_config.py

```diff
@@ -60,20 +62,65 @@ def from_dict(cls, **kwargs: Any) -> "OSDependentLibraryConfig":
 
 
 class GlobalConfig:
-    def __init__(self, global_config_path: str) -> None:
-        with open(global_config_path) as f_config:
-            config_raw = f_config.read()
-        self._is_global_config_yaml = (
-            True if global_config_path.endswith(".yaml") else False
-        )
-        self._content = (
-            yaml_load(config_raw)
-            if self._is_global_config_yaml
-            else json.loads(config_raw)
-        )
+    def __init__(self, global_config_path: str, **kwargs: Any) -> None:
```

Comment:
@hetangmodi-crest this conflicts with https://github.com/splunk/addonfactory-ucc-generator/pull/1655/files, please review that PR first and let's merge it.

I'd like to keep the `__init__` method how it is in that PR.